.github/workflows/build-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+env:
+  DOCKER_ENV_FILE: ".github/workflows/docker.env"
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:12.2-alpine
+        env:
+          POSTGRES_DB: "development"
+          POSTGRES_USER: "dbuser"
+          POSTGRES_PASSWORD: "dbpass"
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+      integresql:
+        image: allaboutapps/integresql:latest
+        env:
+          PGHOST: "postgres"
+          PGUSER: "dbuser"
+          PGPASSWORD: "dbpass"
+      mailhog:
+        image: mailhog/mailhog
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: docker build --target builder --file Dockerfile --tag allaboutapps/go-starter:${GITHUB_SHA:8} .
+      - name: Create container
+        run: docker run -d --env-file $DOCKER_ENV_FILE --network "${{ job.services.postgres.network }}" --name=builder -it allaboutapps/go-starter:${GITHUB_SHA:8}
```

Comment:
We _might_ have to add a `docker stop`/`docker rm` to the end of the pipeline since it looks like the automatic cleanup does not stop the container (since it doesn't track it, I assume) and thus throws a warning when trying to remove the created network.
Not really a pressing issue for us, but for the sake of cleaning up nicely after a run, it might be worth to add.

Reply:
Thank you, will add that to the pipeline.